test/lib/testMayaUsdConverter.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pxr import Gf
+from pxr import Sdf
+from pxr import Usd
+from pxr import Tf
+from pxr import Vt
+
+from mayaUsd import lib as mayaUsdLib
+from maya import cmds
+
+import unittest
+
+class MayaUsdConverterTestCase(unittest.TestCase):
+    """
+    Verify mayaUsd converter.
+    """
+
+    def createStage(self, layerName):
+        layer = Sdf.Layer.CreateAnonymous("Bool")
```

Comment:
Weird name for a layer. ;)

Reply:
So true...I refactored the code to avoid duplication...probably this is how it happened.